packages/ethereum-contracts/ops-scripts/deploy-framework.js

```diff
@@ -563,43 +575,87 @@ module.exports = eval(`(${S.toString()})({skipArgv: true})`)(async function (
                     await superfluid.getSuperTokenFactory.call();
                 if (factoryAddress === ZERO_ADDRESS) return true;
                 const factory = await SuperTokenFactoryLogic.at(factoryAddress);
+                const superTokenLogicAddress =
+                    await factory.getSuperTokenLogic.call();
+                const superTokenLogic = await SuperTokenLogic.at(
+                    superTokenLogicAddress
+                );
+                const constantOutflowNFTLogicAddress =
+                    await superTokenLogic.CONSTANT_OUTFLOW_NFT_LOGIC();
+                const constantInflowNFTLogicAddress =
+                    await superTokenLogic.CONSTANT_INFLOW_NFT_LOGIC();
+                const superTokenFactoryCodeChanged = await codeChanged(
+                    web3,
+                    SuperTokenFactoryLogic,
+                    await superfluid.getSuperTokenFactoryLogic.call(),
+                    [
+                        superfluid.address
+                            .toLowerCase()
+                            .slice(2)
+                            .padStart(64, "0"),
+                    ]
+                );
+                const superTokenLogicCodeChanged = await codeChanged(
```

Comment:
**High**: Note that this is breaking the CI currently. It is always returning true.